tests/performance/local/future_overhead.cpp

```diff
@@ -41,6 +41,10 @@ using hpx::util::high_resolution_timer;
 using hpx::cout;
 using hpx::flush;
 
+#define CDASH_TIMING(name,time) \
+    cout << "<DartMeasurement name=\"" << name << "\" " \
+    << "type=\"numeric/double\"> " << time << " </DartMeasurement> \n";
+
```

Comment:
Could we turn this into a (inline) function (instead of a MACRO), please? Also, I think this could be moved into the header `hpx/util/lightweight_test.hpp`